templates/deployments.yaml

```diff
@@ -1369,6 +1457,10 @@ spec:
             secretKeyRef:
               name: {{ .Release.Name }}-st2-auth
               key: password
+{{- range $env, $value := .Values.st2chatops.env }}
+        - name: {{ $env | quote }}
+          value: {{ $value | quote }}
+{{- end }}
```

Comment:
I think I would drop this, and leave the st2chatops env as is (in `envFrom[1].secretRef` below).
```suggestion
```